ElectronClient/app/gui/ResourceScreen.tsx

```diff
@@ -0,0 +1,180 @@
+import * as React from 'react';
+import { groupBy, orderBy } from 'lodash';
+
+const { connect } = require('react-redux');
+const { _ } = require('lib/locale.js');
+const { themeStyle } = require('../theme.js');
+const { bridge } = require('electron').remote.require('./bridge');
+const { Header } = require('./Header.min.js');
+const prettyBytes = require('pretty-bytes');
+const Resource = require('lib/models/Resource.js');
+const NoteResource = require('lib/models/NoteResource.js');
+
+interface IProps {
+	style: any;
+	theme: any;
+}
```

Comment:
Please don't use Hungarian notation. So just `Props`, and same for the other interfaces.